src/ensembl/src/content/app/browser/browser-tabs/BrowserTabs.tsx

```diff
@@ -22,40 +26,37 @@ const BrowserTabs: FunctionComponent<BrowserTabsProps> = (
       return;
     }
 
+    if (!props.isTrackPanelOpened) {
+      props.toggleTrackPanel(true);
```

Comment:
If the tabs are already hidden when then TrackPanel is closed, how is this onClick getting called?

Reply:
Tabs aren't always hidden on a large screen when the track panel is closed. So this listener can still be fired even when the track panel is closed.

Reply:
Cool 👍